src/app/query/common/remote-config/remote-config.query.ts

```diff
@@ -35,3 +39,27 @@ export function useConfigBitcoinSendEnabled() {
     software: config?.bitcoinSendEnabled ?? true,
   });
 }
+
+export function useConfigSbtc() {
+  const config = useRemoteConfig();
+  const sbtc = config?.sbtc;
+  const network = useCurrentNetwork();
+
+  return useMemo(() => {
+    const displayPromoCardOnNetworks = (sbtc as any)?.showPromoLinkOnNetworks ?? [];
```

Comment:
Do we have to cast as any? I made a comment I deleted thinking this was for all config properties, but I see below we can have some type safety on `sbtc.enabled` - would love to see that here too for refactorability and not surprising ourselves with small avoidable bugs in the future.